debian/changelog

```diff
@@ -0,0 +1,5 @@
+libplayback (0.5-21+0m5) unstable; urgency=low
```

Comment:
Last version in Maemo is 0_0.5-23+0m5. So you should have version higher then this.